packages/core-data/src/queried-data/selectors.js

```diff
@@ -104,7 +103,7 @@ function getQueriedItemsUncached( state, query ) {
  *
  * @return {?Array} Query items.
  */
-export const getQueriedItems = createSelector( ( state, query = {} ) => {
+export const getQueriedItems = ( state, query = {} ) => {
```

Comment:
I know this is still a draft, but I'm curious why the `createSelector` was removed from here :) was this intentional?